### DISCUSSION TITLE
Make packages/apps part of the changelog within a release

### DISCUSSION THREAD

**Discussion by AnonymousWP** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#329] [Polls]
We initially discussed this question internally, but the opinions differed, so I decided to ask you - the community - for your opinion. Users are the majority that will read a changelog of in this case: UAD-ng.

If you remember, the OG UAD project and we used to show which packages between releases have been added/modified. For an example, see the following screenshot:

![image](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/assets/50231698/a94b3707-e25b-469c-bbc5-a2547823f530)

Two major considerations:

1. Downside is that the changelog may look cluttered / grows to a large size in case packages are added
2. Benefit is that users will know which new packages have been added without having to filter the merged PRs

**Comment by libreom** [comment]
Yes

**Comment by AnonymousWP** [comment]
Closing this poll as it probably reached its peak in terms of voting. This discussion has now been solved by https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/311/commits/befb529fc7057bc4880a5a8d3a15552cd9d3940f.
